### PR TITLE
Reboot update module

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: reboot/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Install the Update Module
+
+Download the latest version of this Update Module by running:
+
+```
+mkdir -p /usr/share/mender/modules/v3 && wget -P /usr/share/mender/modules/v3 https://raw.githubusercontent.com/mendersoftware/mender-update-modules/master/reboot/module/reboot && chmod +x /usr/share/mender/modules/v3/reboot
+```
+
+This Update Module depends on the `kubectl` CLI.
```
Comment (drewmoseley) [inline reboot/README.md:29]: Is this a copy/paste thing or does it really require kubectl?

File: reboot/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* something went wrong and you have to reboot your device remotely, and all
+access to the device you have is Mender.
+
+### Specification
+
+|||
+| --- | --- |
+|Module name| reboot |
+|Supports rollback|no|
+|Requires restart|no|
```
Comment (drewmoseley) [inline reboot/README.md:16]: Should this be "yes"?

Comment (merlin-northern) [reply reboot/README.md]: wow. of course. thanks.

File: reboot/README.md (hunk 1 above)
Comment (merlin-northern) [reply reboot/README.md]: ah left that one. copy-paste. thanks.

File: reboot/reboot-gen [hunk 3]
```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
```
Comment (mirzak) [inline reboot/reboot-gen:1]: Late to the game but it seems a bit excessive to have a generator script when all you need to do is:

```
DEVICE_TYPE="raspberrypi4"
mender-artifact write module-image \
    -T reboot \
    -t ${DEVICE_TYPE} \
    -o reboot.mender \
    -n reboot
```

**Comment by mirzak** [comment]
Also remember to add an post for it here https://hub.mender.io/c/update-modules/13

File: reboot/reboot-gen (hunk 3 above)
Comment (merlin-northern) [reply reboot/reboot-gen:1]: Thank you Mr. Krak. I thought it was a standard to provide a script. I can remove it if you want me to. Actually you will have to make me removeit.

**Comment by merlin-northern** [comment]
> Also remember to add an post for it here https://hub.mender.io/c/update-modules/13

I hope you are happy now: https://hub.mender.io/t/reboot-remotely/2247

**Comment by mirzak** [comment]
> I hope you are happy now: https://hub.mender.io/t/reboot-remotely/2247

Very :)

**Comment by merlin-northern** [comment]
> > I hope you are happy now: https://hub.mender.io/t/reboot-remotely/2247
> 
> Very :)

making you happy is one of those rare pleasures I enjoy here.

File: reboot/reboot-gen (hunk 3 above)
Comment (mirzak) [reply reboot/reboot-gen:1]: Yeah, somewhat standard for the more "complex" modules that require input files etc. In this case `mender-artifact` alone should work just fine as there is no additional error checking beside what is already provided by the `mender-artifact` tool.

Less code is always better. This is is not a priority, but if you wish to clean it up at some point I would be happy :)

Comment (kacf) [reply reboot/reboot-gen:1]: My vote would be to keep it, actually. I know it's kind of redundant, but I like that we have one, and only one, way to generate Update Module Artifacts. Easier to wrap your head around if you're new to the game.

Comment (merlin-northern) [reply reboot/reboot-gen:1]: I would love to keep you both happy, but if I have to vote I would keep it as well. makes sense to me what you say.

Comment (mirzak) [reply reboot/reboot-gen:1]: Ok, lets keep it then :+1:

**Pull request merged**